dev/spec-setup.js

```diff
@@ -1,10 +1,15 @@
 /* eslint-disable global-require, no-console */
-
 var rowdy = require("rowdy");
 var MochaAdapter = rowdy.adapters.mocha;
 
+var SERVER_HOST = "127.0.0.1";
+var SERVER_PORT = "3000";
+global.TEST_FUNC_BASE_DIR = process.env.TEST_FUNC_BASE_DIR || "";
+global.TEST_FUNC_BASE_URL = process.env.TEST_FUNC_BASE_URL || "http://" + SERVER_HOST + ":" + SERVER_PORT + global.TEST_FUNC_BASE_DIR;
+
 var config = require("rowdy/config");
 config.options.driverLib = "webdriverio";
+config.settings.local.default.remote = { baseUrl: global.TEST_FUNC_BASE_URL };
```

Comment:
We're on the same page there!

Here's the `baseUrl` option I'm passing into `webdriverio.remote(opts)` (which is maybe just a poor API on their part)
http://webdriver.io/guide/getstarted/configuration.html#baseUrl

Which all it does is prefix every url command (`client.url('/some/url')`) with the `baseUrl` passed into that `remote` call.

https://github.com/webdriverio/webdriverio/blob/master/lib/protocol/url.js#L28